### PR TITLE
Update jupyter dependencies to latest

### LINKED ISSUE #141: Some dashboards not working in Safari
Simple dashboards are working fine in Safari, but the plotting demo and resizable widget demo do not.

There are no obvious relevant errors in Javascript console.

Steps to complete:

* [ ] Move to ipywidgets 5.1.x + declarative widgets 0.5.2
* [ ] Either go with #108 or move to latest jupyter-js-notebook
* [ ] Either make rendermime's HTML transform work in Safari or replace it here with one that does

### PR DESCRIPTION
**Pull request by jhpedemonte**
Also contains fix for running scripts on Safari.

Fixes #141

### CODE REVIEW

**Comment by jhpedemonte** [comment]
Hmm, integration-test fails:

```
  1) dashboard with widgets "before all" hook:
     Uncaught Error: connect ECONNREFUSED 192.168.99.100:3000
      at Object.exports._errnoException (util.js:856:11)
      at exports._exceptionWithHostPort (util.js:879:20)
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1062:14)
```

Why would it refuse the connection?

**Comment by jhpedemonte** [comment]
Forgot to include `commonmark` npm module, therefore dashboard server was failing to start as part of integration tests. Now everything passes. Good to go.

File: client/less/notebook.less [hunk 1]
```diff
@@ -75,6 +75,10 @@
 
     .p-Widget {
         padding: 0.4em;
+
+        .jp-OutputArea-prompt {
+            display: none;
```
Comment (parente) [inline client/less/notebook.less]: So the prompts are still in there, just hidden?

**Comment by parente** [comment]
`make clean examples kill build dev-install dev`. Every example notebook I tested worked, including the plotting one in Safari. I tested some additional ones without any major issues as well.

One new annoying behavior that I'm going to open as a follow-on issue. Every time I open a paper-dropdown, the entire browser page shifts to the right. It shifts back when I close it. It happens in every demo that uses the dropdown, and in both Chrome and Safari.

File: client/less/notebook.less (hunk 1 above)
Comment (jhpedemonte) [inline client/less/notebook.less]: Right. That's written by the notebook's output area, so we can only hide it.

**Pull request merged**